packages/@sanity/core/src/commands/dataset/alias/deleteAliasHandler.js

```diff
@@ -0,0 +1,42 @@
+import validateDatasetAliasName from '../../../actions/dataset/alias/validateDatasetAliasName'
+import * as aliasClient from './datasetAliasesClient'
+import {ALIAS_PREFIX} from './datasetAliasesClient'
+
+export default async (args, context) => {
+  const {apiClient, prompt, output} = context
+  const [, ds] = args.argsWithoutOptions
```

Comment:
Perhaps we could use a more descriptive name for `ds`? 🙂

Reply:
Ds is used almost everywhere in the cli just followed the maiming.